specs/integration/search/CreateDatasetData.js

```diff
@@ -0,0 +1,2 @@
+datasetName = "Test dataset " + Math.random().toString(36).substring(7)
+datasetDescription = "Test dataset description"
```

Comment:
Rather use `datasetName + " description"`, it can help in cleaning the database if something goes awry in tests.